lua/lvim/core/nvimtree.lua

```diff
@@ -118,19 +117,16 @@ function M.setup()
     vim.g.netrw_banner = false
   end
 
+  -- Add useful keymaps
   local tree_cb = nvim_tree_config.nvim_tree_callback
-
-  if not lvim.builtin.nvimtree.setup.view.mappings.list then
-    lvim.builtin.nvimtree.setup.view.mappings.list = {
-      { key = { "l", "<CR>", "o" }, cb = tree_cb "edit" },
-      { key = "h", cb = tree_cb "close_node" },
-      { key = "v", cb = tree_cb "vsplit" },
-    }
-  end
-
-  local tree_view = require "nvim-tree.view"
+  vim.list_extend(lvim.builtin.nvimtree.setup.view.mappings.list, {
+    { key = { "l", "<CR>", "o" }, cb = tree_cb "edit" },
+    { key = "h", cb = tree_cb "close_node" },
+    { key = "v", cb = tree_cb "vsplit" },
+  })
```

Comment:
This will hard-code the values, we should keep the condition but fix it first:
```suggestion
  if #lvim.builtin.nvimtree.setup.view.mappings.list == 0 then
    vim.list_extend(lvim.builtin.nvimtree.setup.view.mappings.list, {
      { key = { "l", "<CR>", "o" }, cb = tree_cb "edit" },
      { key = "h", cb = tree_cb "close_node" },
      { key = "v", cb = tree_cb "vsplit" },
    })
  end
```